src/components/input-dropdown/input-dropdown.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="ux-select-container">
+    <button #button type="button" class="form-control"
+            [uxMenuTriggerFor]="menu">
+        <ng-template #defaultDisplayContent>{{selectedString()}}</ng-template>
+        <ng-container [ngTemplateOutlet]="displayContentRef ? displayContentRef : defaultDisplayContent"></ng-container>
+    </button>
+    <div class="ux-select-icons">
+        <ux-icon name="close"
+                 class="ux-select-icon ux-select-clear-icon"
+                 *ngIf="allowNull && selected"
+                 (click)="resetValue($event)">
+        </ux-icon>
+        <ux-icon name="chevron-down"
+                 class="ux-select-icon ux-select-chevron-icon"
+                 (click)="menuTrigger.toggleMenu(); $event.stopPropagation()">
+        </ux-icon>
+    </div>
+</div>
+
+<ux-menu #menu menuClass="select-menu"
+         (opened)="filterInputElement.nativeElement.focus()">
+
+    <div [style.max-height.px]="maxHeight"
+         [style.width.px]="button.offsetWidth">
+
+        <div *ngIf="!hideFilter"
+             class="filter-container">
+
+            <input #filterInput
+                    type="text"
+                    [placeholder]="placeholder"
+                    class="form-control"
+                    [(ngModel)]="filterText"
+                    (input)="filterChange.emit(filterText)"
+                    (click)="$event.stopPropagation()" />
+
+            <button type="button"
+                    class="btn btn-flat btn-icon-only filter-button"
+                    (click)="resetFilter($event)"
+                    [tabindex]="filterText.length > 0 ? 0 : 1">
```

Comment:
We try to avoid positive tabindexes. If you want to prevent tabbing change it to `-1` instead of `1`

Reply:
That was a typo. It is fixed.